hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/client/ClientProtocolStub.java

```diff
@@ -642,6 +642,13 @@ public String createSnapshot(String volumeName,
     return "";
   }
 
+  @Override
+  public void renameSnapshot(String volumeName, String bucketName,
+      String fromSnapshotName, String toSnapshotName)
```

Comment:
nit: to align with other places.

```suggestion
      String snapshotOldName, String snapshotNewName)
```